server/build/docker/keycloak/README.md

```diff
@@ -18,44 +18,12 @@ To use this keycloak image, we suggest you to use this configuration settings:
 - First Name Attribute: `firstName`
 - Last Name Attribute: `lastName`
```

Comment:
Should `make config-saml` also set these fields so it's easier to set up the connection between Mattermost and Keycloak?

Reply:
The only problem is ldap has attributes of the same name so those would also get overwritten. I'll put the correct values in this README so people know